src/js/common/components/LoginModal.jsx

```diff
@@ -57,7 +57,12 @@ class LoginModal extends React.Component {
     const loginUrl = this.props.user.login.loginUrl;
     const onUpdateLoginUrl = this.props.onUpdateLoginUrl;
     this.loginUrlRequest = handleLogin(loginUrl, onUpdateLoginUrl);
-    this.loginUrlRequest.then(() => this.props.onClose());
+    this.loginUrlRequest.then(() => {
```

Comment:
Yeah, that's more what I'm looking for, but I'd rather not return the promise from `handleLogin()` if I don't have to. If the test I wrote fails either consistently or even just spottily (spottily...spottily...spell checker says it's a word!), I'll rewrite it like that.

Also, I think I may start using "howdy" for debugging now. 😄